### PR TITLE
Updated backend to create orders as xlsx and update email script to p…

### PR DESCRIPTION
**Pull request by jacob-butler**
…ull xlsx from email

### CODE REVIEW

File: backend/app/write_order_xlsx.py [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+                pack_cell.value = "UPC"
+                pack_cell.font = medium_font
+            pack_cell.alignment = center
+    for col in [5,10]:
+        case_cell = worksheet.cell(column = col, row = 2)
+        case_cell.value = "CASE"
+        case_cell.font = small_font
+        case_cell.alignment = center
+
+def write_catagories(worksheet, order_info):
```
Comment (DuaneShaffer) [inline backend/app/write_order_xlsx.py]: category

File: backend/app/write_order_xlsx.py [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+        worksheet.column_dimensions[get_column_letter(column_cells[0].column)].width = length
+    
+    ## give both product columns the same width
+    prod_len = max(col2_length,col7_length)
+    worksheet.column_dimensions[get_column_letter(2)].width = prod_len
+    worksheet.column_dimensions[get_column_letter(7)].width = prod_len
+
+    ## set scaling for to fit page and left and right margins to zero
+    worksheet.sheet_properties.pageSetUpPr.fitToPage = True
+    worksheet.page_setup.fitToWidth = True
```
Comment (DuaneShaffer) [inline backend/app/write_order_xlsx.py]: Weren't we talking about this being =false?

Comment (jacob-butler) [reply backend/app/write_order_xlsx.py]: I tested leaving out the `worksheet.sheet_properties.pageSetUpPr.fitToPage = True`, line and it did not format the page as desired.

File: backend/app/write_order_xlsx.py (hunk 1 above)
Comment (jacob-butler) [reply backend/app/write_order_xlsx.py]: fixed

**Pull request merged**